bridge/docs/MP-268:BridgePoolDeployment.puml

```diff
@@ -0,0 +1,21 @@
+@startuml
```

Comment:
Would be nice to have a PNG/SVG of this checked in as well for easier reference and shouldn't add much size.